pkg/server/registry/apigroups/acorn/apps/validator.go

```diff
@@ -419,6 +424,23 @@ func (s *Validator) checkScheduling(ctx context.Context, params *apiv1.App, proj
 	return validationErrors
 }
 
+func getClassForWorkload(computeClassList *apiv1.ComputeClassList, computeClasses v1.ComputeClassMap, container v1.Container, workload string) (*apiv1.ComputeClass, error) {
+	wc := computeclasses.GetWorkloadClassNameForWorkload(workload, container, computeClasses)
```

Comment:
wc is leftover from when this was called WorkloadClass.
```suggestion
	cc := computeclasses.GetWorkloadClassNameForWorkload(workload, container, computeClasses)
```